sdk-extensions/autoconfigure/src/testFullConfig/java/io/opentelemetry/sdk/autoconfigure/FileConfigurationTest.java

```diff
@@ -175,4 +180,27 @@ void configFile_Error(@TempDir Path tempDir) throws IOException {
         .isInstanceOf(ConfigurationException.class)
         .hasMessage("Unrecognized span exporter(s): [foo]");
   }
+
+  @Test
+  void configFile_ExtendedConfigProperties() {
+    ConfigProperties config =
+        DefaultConfigProperties.createFromMap(
+            Collections.singletonMap("OTEL_CONFIG_FILE", configFilePath.toString()));
```

Comment:
We need to provide a way to deactivate `OTEL_CONFIG_FILE` config loading if the properties supplier is being used directly in the Autoconfiguration.

Reply:
This is unrelated to this PR. Its already possible to deactivate `OTEL_CONFIG_FILE` via property suppliers. See the [source code](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java#L387) for how `OTEL_CONFIG_FILE` is loaded. Its accessed via `ConfigProperties.getString("otel.config.file")` using a `ConfigProperties` instance that is resolved using all the property suppliers and customizers typical in autoconfigure.